### ISSUE TITLE
Feature Implementation Suggestion

### ISSUE THREAD

**Issue by RHQ-Rusty** [lovvskillz/python-discord-webhook#13]
I've created a simple-to-use, easy-to-follow CLI for using this tool to fully customize and send content to a user-definable webhook URL. It also supports the full capacity to save the webhookURL with AES Encryption. A [link to download it](https://github.com/RHQOnline/Discord-Webhook-Utility/blob/master/README.md) can be found here, as well as usage instructions; try it out and let me know how it works. If you want, I can send in the source code and it can be added directly into the repo.

Features
- Fully Functional
- Saving and Encrypting of stored Webhook(s)
- Custom Usernames per Message 
- Fully Customizable Embeds and Content (Fields, Authors, Images, Thumbnails, etc.)

Some Images of the CLI Applet
![](https://i.imgur.com/cPAoFOr.png)
![](https://i.imgur.com/HztcaMS.png)

Some Images of it's Output
![](https://i.imgur.com/yJKyr2J.png)
![](https://i.imgur.com/C8ByEZ4.png)

**Comment by RHQ-Rusty** [comment]
@lovvskillz

**Comment by lovvskillz** [comment]
Hi,
thank you for your effort. But I still don't understand the purpose of sending a webhook via the CLI.

**Comment by RHQ-Rusty** [comment]
> Hi,
> thank you for your effort. But I still don't understand the purpose of sending a webhook via the CLI.

I made it for a few communities that utilize Discord to send announcements. It's a GUI-CLI program that is aimed around making the sending of content to Discord Webhook URLs much easier, and it's done in Python and it's essentially built off of the python-discord-webhook library. I just figured it'd be useful for those who want a program to send customizable input to a webhook utilizing this library.
